trees_emit.h

```diff
@@ -21,7 +21,7 @@ extern ZLIB_INTERNAL const int base_dist[D_CODES];
 /* Bit buffer and deflate code stderr tracing */
 #ifdef ZLIB_DEBUG
 #  define send_bits_trace(s, value, length) { \
-        Tracevv((stderr, " l %2d v %4llx ", length, value)); \
+        Tracevv((stderr, " l %2d v %4llx ", length, (unsigned long long)value)); \
```

Comment:
Can we use `PRId32` and `PRId64` here? Might have to include inttypes.h..

Reply:
Do you mean we should use them instead of the cast? I'd rather not do that: since this is a macro, we don't know the types of length and value.

Reply:
Hm, this actually means we'd better cast length too, and put both of them in parenthesis.